src/views/Home/components/About/AboutBox/AboutBox.scss

```diff
@@ -17,11 +18,20 @@
         padding: 0 2% 0 2%;
         border-top-left-radius: 33px;
         border-top-right-radius: 33px;
-        @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 80px);
+        @include responsiveness.font($responsive: 3.8vw, $min: 35px, $max: 80px);
         @include responsiveness.screen(smallest, phone) {
-            @include responsiveness.font($responsive: 7.1vw, $min: 22px, $max: 40px);
+            @include responsiveness.font($responsive: 7.1vw, $min: 20px, $max: 40px);
+            text-align: center;
             padding: 0 26px 0 30px;
         }
+        @include responsiveness.screen(smallest) {
+            padding: 0 0 0 0px;
```

Comment:
Also, this is just `padding: 0`, right?

Reply:
Oh yeah this was probably me just copying from dev tools, I'll update this!